components/api_version/src/api_v2.rs

```diff
@@ -36,6 +37,14 @@ bitflags::bitflags! {
     }
 }
 
+fn infinite_end_of(prefix: &[u8]) -> [u8; 4] {
+    debug_assert!(prefix.len() >= KEY_PREFIX_SIZE);
+
+    let end_bytes = [prefix[0], prefix[1], prefix[2], prefix[3]];
```

Comment:
This operation will be often used later, and it can be encapsulated as a special function to obtain the first four bits of the array.

Reply:
Make sense.